### PR TITLE
chore(open-payments): fetch OpenAPI specs locally

### PR DESCRIPTION
**Pull request by wilsonianb**
## Changes proposed in this pull request

- Fetch OpenAPI specs locally in `open-payments`
- Symlink necessary OpenAPI specs in `auth` and `backend`
- `pnpm fetch-schemas` is no longer required, and Open Payments OpenAPI specs now live in a single place in the monorepo (where they can optionally be modified).

## Context

- partially reverts #799
- setup for https://github.com/interledger/rafiki/issues/844

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

File: packages/open-payments/package.json [hunk 1]
```diff
@@ -8,11 +8,15 @@
   ],
   "scripts": {
     "build:deps": "pnpm --filter openapi build",
-    "build": "pnpm build:deps && pnpm clean && tsc --build tsconfig.json",
+    "build": "pnpm build:deps && pnpm clean && tsc --build tsconfig.json && pnpm copy-files",
     "clean": "rm -fr dist/",
-    "generate:types": "npx ts-node scripts/generate-types.ts",
+    "fetch-schemas": "./scripts/fetch-schemas.sh",
+    "generate:auth-server-types": "openapi-typescript src/openapi/auth-server.yaml --output src/openapi/generated/auth-server-types.ts -t",
```
Comment (mkurapov) [inline packages/open-payments/package.json:14]: why types over interfaces btw?

**Review by mkurapov — APPROVED** [review]
This is great, keeps it in one place

File: README.md [hunk 2]
```diff
@@ -67,9 +67,6 @@ pnpm clean
 
 # install dependencies
 pnpm i
-
-# pull in latest openapi specs:
-pnpm fetch-schemas
```
Comment (mkurapov) [inline README.md:72]: I think the README in the `auth` package also has a `fetch-schemas` reference

File: packages/open-payments/package.json (hunk 1 above)
Comment (wilsonianb) [reply packages/open-payments/package.json:14]: I don't know :sweat_smile: 
`-t` isn't even an option in the version we're using...
```
  Generate TypeScript types from Swagger OpenAPI specs

  Usage
    $ openapi-typescript [input] [options]

  Options
    --help                       display this
    --output, -o                 Specify output file (default: stdout)
    --auth                       (optional) Provide an authentication token for private URL
    --headersObject, -h          (optional) Provide a JSON object as string of HTTP headers for remote schema request
    --header, -x                 (optional) Provide an array of or singular headers as an alternative to a JSON object. Each header must follow the key: value pattern
    --httpMethod, -m             (optional) Provide the HTTP Verb/Method for fetching a schema from a remote URL
    --immutable-types, -it       (optional) Generates immutable types (readonly properties and readonly array)
    --additional-properties, -ap (optional) Allow arbitrary properties for all schema objects without "additionalProperties: false"
    --default-non-nullable       (optional) If a schema object has a default value set, don’t mark it as nullable
    --prettier-config, -c        (optional) specify path to Prettier config file
    --raw-schema                 (optional) Parse as partial schema (raw components)
    --version                    (optional) Force schema parsing version
```

**Review by mkurapov — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -17,12 +17,11 @@
     "check:prettier": "prettier --check .",
     "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
     "build": "tsc --build",
-    "fetch-schemas": "pnpm --filter auth --filter backend fetch-schemas",
     "localenv": "docker compose -f ./infrastructure/local/docker-compose.yml -f ./infrastructure/local/peer-docker-compose.yml",
     "localenv:build": "docker compose -f ./infrastructure/local/docker-compose.yml -f ./infrastructure/local/peer-docker-compose.yml -f ./infrastructure/local/build-override.yml build",
     "localenv:seed:auth": "pnpm -C ./packages/auth knex seed:run --env=development && pnpm -C ./packages/auth knex seed:run --env=peerdevelopment",
     "localenv:dbvolumes:remove": "docker volume rm local_database-data && docker volume rm local_tigerbeetle-data",
-    "sanity": "pnpm fetch-schemas pnpm -r build && pnpm -r test"
```
Comment (mkurapov) [inline package.json:25]: What was the `sanity` for, originally? Should anything else belong here?

**Review by mkurapov — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (wilsonianb) [reply package.json:25]: I don't know why it was added
- https://github.com/interledger/rafiki/pull/458

**Pull request merged**